content/06.model-complexity.md

```diff
@@ -1,34 +1,34 @@
-### Manage model complexity while preserving the value of machine learning
+### Augmenting machine learning models with statistical techniques to reduce misinterpretation
 
 
-<!-- TODO: stick to one example method of each approach so that we get to describe why we need to use it --->
-
 Machine learning methods are generally accompanied by a few critical assumptions. 
-Assumption 1, the dataset contains equal number of samples for all classes in a dataset (i.e. no imbalance among classes). 
-2. the dataset is complete with adequate samples for all features (lack of sparsity).
-3. there is no ambiguity about the labels for the samples in the dataset (i.e. no label-noise, applicable for supervised learning methods).
+First, ML methods often work best on datasets that contain equal number of samples for all categories (no "class imbalance").
+Second, the dataset is complete; all samples have measurements for all variables in the dataset (i.e., the dataset is not "sparse", meaning that it is not missing data for some samples).
+Third, there is no ambiguity about the labels for the samples in the dataset (no "label-noise", meaning that, for example, we are confident in the disease of each patient in a dataset).  
```

Comment:
Do we need this much explanation here in light of what I quoted from the intro in https://github.com/jaybee84/ml-in-rd/pull/198/files#r800215869?